src/pure-function-helpers/web-message-type.ts

```diff
@@ -52,7 +52,9 @@ export function webMessageType (
      */
     case WebMessageType.SYS:
       return MessageType.Text
-
+    // add recall type
```

Comment:
I think we don't need the comment here, since the type is clearly stated it's purpose.